pkg/apis/core/validation/validation.go

```diff
@@ -2195,6 +2199,17 @@ func ValidatePersistentVolumeClaimSpec(spec *core.PersistentVolumeClaimSpec, fld
 				"must match dataSourceRef"))
 		}
 	}
+	if spec.VolumeAttributesClassName != nil {
+		if opts.EnableVolumeAttributesClass {
+			if len(*spec.VolumeAttributesClassName) == 0 {
+				allErrs = append(allErrs, field.Forbidden(fldPath.Child("volumeAttributesClassName"), "an empty string is disallowed"))
+			} else {
+				for _, msg := range ValidateClassName(*spec.VolumeAttributesClassName, false) {
```

Comment:
@sunnylovestiramisu 

reuse the `ValidateClassName ` or define a new func `ValidateVolumeAttributesClassName`?

Reply:
I think there is no special validation for VAC's name so it is fine to reuse, and I cannot see for the future there will be any special naming around VAC. What do you think?

Reply:
reuse +1